DataFormats/Phase2TrackerDigi/src/classes_def.xml

```diff
@@ -5,12 +5,20 @@
    <class name="edm::Ref< edm::DetSetVector< Phase2TrackerDigi >, Phase2TrackerDigi >"/>
    <class name="std::vector< edm::Ref< edm::DetSetVector< Phase2TrackerDigi >, Phase2TrackerDigi > >"/>
    <class name="edm::DetSet<Phase2TrackerDigi>"/>
+   <class name="edm::DetSet<Phase2ITQCore>"/>
+   <class name="edm::DetSet<Phase2ITChipBitStream>"/>
```

Comment:
Hi @makortel Thank you for very constructive comments. I have some trouble understanding only this one. I understood what you mean but I am not sure how these info gets retrieved. Like, I can build it, but in order to understand the "version" and the "checksum", what/where should i look at ?

Reply:
One way would be to add e.g.
```xml
   <class name="Phase2ITQCore" ClassVersion="3">
    <version ClassVersion="3" checksum="0"/>
   </class>
   <class name="Phase2ITChipBitStream" ClassVersion="3">
    <version ClassVersion="3" checksum="0"/>
   </class>
```
above (after the `Phase2TrackerDigi` `</class>` element) and run `scram b`. The compilation will fail because the checksums are wrong, but the correct checksums are in the error messages (ignore the request to increase class version to 4).

For more information see https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning